polkadot/node/subsystem-bench/docker/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+services:
+  grafana:
+    image: grafana/grafana-enterprise:latest
```

Comment:
nit: Shouldn't we use an explicit version here?

Reply:
Why? I think that yml is going to be updated much rarer than the grafana (and others) version. And it's only for local development, if someone has particular preferences they can change it locally.

Reply:
With `latest` you have no idea what version was used for a deployment, which is bad from reproducibility point of view. Also if a breaking change is introduced in any of the images we might run into issues.

But if you say it's just for local development and we don't care for the above - fine by me :)